packages/keycodes/src/index.ts

```diff
@@ -240,6 +240,19 @@ export const rawShortcut: WPModifierHandler< WPKeyHandler< string > > =
 		};
 	} );
 
+/**
+ * An object that contains functions to get shortcuts in a format compatible
+ * with the [`aria-keyshortcuts` HTML attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-keyshortcuts).
+ *
+ * @example
+ * ```js
+ * // Assuming macOS:
+ * ariaKeyShortcut.primary( 'c' )
+ * // "meta+c"
+ * ```
+ */
+export const ariaKeyShortcut = rawShortcut;
```

Comment:
```suggestion
export const ariaKeyshortcut = rawShortcut;
```

Should `Keyshortcut` be one word to match the ARIA property? 🤔 (No strong opinion, ARIA names don't really delimit words anyway, e.g. `describedby`).

Reply:
The JS property is `ariaKeyShortcuts`, so I think this is fine. https://developer.mozilla.org/en-US/docs/Web/API/Element/ariaKeyShortcuts